deploy/roles/ultra-edit/defaults/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+ue_repo_name_version_16_18: UltraEdit-16.1-0.18.x86_64
+ue_repo_name_version_3_3: UltraEdit-3.3-0.12.x86_64
+ue_repo_name_version_16_22: UltraEdit-16.1-0.22.x86_64
+ue_repo_name_version_15_8: UltraEdit-15.1-0.8.x86_64
```

Comment:
@stirlingv It might make sense to have this be a single list variable instead of 4 separate variables, called `ue_uninstall_list` or something, then just do `with_items: {{ ue_uninstall_list }}` below.